### PR TITLE
[CI] Add e2e test for /settings > Environment

### PR DESCRIPTION
**Pull request by ramrodo**
Signed-off-by: Rodolfo Martínez Vega <rodomar@outlook.com>

**Description**

This PR adds an e2e test for `/setttings` route. Specifically it:
- Creates a Kind cluster
- Loads and tests Kind cluster configuration into Meshery UI by pinging it and validating the response

This PR is related to #1037

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
@ramrodo is this failure expected at the moment? https://github.com/layer5io/meshery/pull/2139/checks?check_run_id=1602928307#step:10:233

**Comment by ramrodo** [comment]
> @ramrodo is this failure expected at the moment? https://github.com/layer5io/meshery/pull/2139/checks?check_run_id=1602928307#step:10:233

@leecalcote yes. It is a WIP. I'm testing some tests.

**Comment by leecalcote** [comment]
> > @ramrodo is this failure expected at the moment? https://github.com/layer5io/meshery/pull/2139/checks?check_run_id=1602928307#step:10:233
> 
> @leecalcote yes. It is a WIP. I'm testing some tests.

Oh, very good. Sounds great.

**Comment by ramrodo** [comment]
@leecalcote this PR is ready to be reviewed

**Review by leecalcote — DISMISSED** [review]
Wow. Very cool, @ramrodo. Pretty neat test.

File: .github/workflows/ci.yml
Comment (kushthedude) [inline .github/workflows/ci.yml:193]: Can we use this https://github.com/helm/kind-action#example-workflow ?

**Review by kushthedude — APPROVED** [review]
None of the error here is due to your PR @ramrodo , there lint errors already in models and main.go.

Great work here @ramrodo !

**Comment by ramrodo** [comment]
Thanks for reviewing this @kushthedude. But I'll wait until we fix those CI errors before merging this PR. We should not merge any PR if the tests are failing.

**Comment by ramrodo** [comment]
@leecalcote @kushthedude the tests finally pass. Could you review this PR again?

**Review by kushthedude — APPROVED** [review]

**Comment by kushthedude** [comment]
Good to see all green again!

**Comment by leecalcote** [comment]
Green!

**Pull request merged**
